app/components/npq_separation/navigation_structures/guidance_navigation_structure.rb

```diff
@@ -25,7 +25,12 @@ def structure
           Node.new(
             name: "How-to guides",
             href: api_guidance_page_path(page: "/", anchor: "how-to-guides"),
-            prefix: "/api/guidance/",
+            prefix: "/api/guidance//#how-to-guides",
+          ) => [],
+          Node.new(
+            name: "Process diagrams",
+            href: api_guidance_page_path(page: "/", anchor: "process-diagrams"),
+            prefix: "/api/guidance/participant-training-journey-diagrams",
```

Comment:
process diagrams will eventually have multiple pages similar to how to guides, can we make sure it allows that rather than tie it to the first page?